### ISSUE TITLE
Data for experiments

### ISSUE THREAD

**Issue by BillyZhaohengLi** [illinoisdata/kishu#61]
Download data into `/data/elastic-notebook/data` for testing.

List of current notebooks (tick once data is downloaded):
- [ ] [XGBoost Starter - [0.793]](https://www.kaggle.com/code/cdeotte/xgboost-starter-0-793)
- [ ] [Approaching (Almost) Any NLP Problem on Kaggle](https://www.kaggle.com/code/abhishek/approaching-almost-any-nlp-problem-on-kaggle)
- [x] [MSCI EDA which makes sense](https://www.kaggle.com/code/ambrosm/msci-eda-which-makes-sense)
- [ ] [NBME / Deberta-base baseline [train]](https://www.kaggle.com/code/yasufuminakama/nbme-deberta-base-baseline-train)
- [ ] [Scaling Up: Road to the Top, Part 3](https://www.kaggle.com/code/jhoward/scaling-up-road-to-the-top-part-3)
- [ ] [GCViT: Global Context Vision Transformer](https://www.kaggle.com/code/awsaf49/gcvit-global-context-vision-transformer)
- [x] [AI4Code - Language Detection and Model Tuning](https://www.kaggle.com/code/andradaolteanu/ai4code-language-detection-and-model-tuning)
- [x] [Visual In-Depth EDA – VinBigData Competition Data](https://www.kaggle.com/code/dschettler8845/visual-in-depth-eda-vinbigdata-competition-data)
- [x] [Time Series forecasting with Prophet](https://www.kaggle.com/code/robikscube/time-series-forecasting-with-prophet)
- [x] [AI4Code Detailed EDA](https://www.kaggle.com/code/odins0n/ai4code-detailed-eda)
- [ ] [Sartorius - Starter Torch Mask R-CNN [LB=0.273]](https://www.kaggle.com/code/julian3833/sartorius-starter-torch-mask-r-cnn-lb-0-273)
- [ ] [SWA, Apex AMP & Interpreting Transformers in Torch](https://www.kaggle.com/code/rhtsingh/swa-apex-amp-interpreting-transformers-in-torch)
- [ ] [YoloX training pipeline COTS dataset [LB 0.507] !!](https://www.kaggle.com/code/remekkinas/yolox-training-pipeline-cots-dataset-lb-0-507)
- [x] [Bruteforce Clustering](https://www.kaggle.com/code/thedevastator/bruteforce-clustering)
- [ ] [DevFest2020IdentifyingMelanomaInLesionImages](https://www.kaggle.com/code/usharengaraju/devfest2020identifyingmelanomainlesionimages/comments)
- [ ] [NLP - EDA, Bag of Words, TF IDF, GloVe, BERT](https://www.kaggle.com/code/vbmokin/nlp-eda-bag-of-words-tf-idf-glove-bert)
- [ ] [Deep Learning For NLP: Zero To Transformers & BERT](https://www.kaggle.com/code/tanulsingh077/deep-learning-for-nlp-zero-to-transformers-bert)
- [ ] [Segmentation in PyTorch using convenient tools](https://www.kaggle.com/code/artgor/segmentation-in-pytorch-using-convenient-tools)
- [ ] [Plant Pathology 2020 : EDA + Models](https://www.kaggle.com/code/tarunpaparaju/plant-pathology-2020-eda-models)
- [ ] [A look at different embeddings.!](https://www.kaggle.com/code/sudalairajkumar/a-look-at-different-embeddings/notebook)

Deprecated:
- [x] [NFL Data bowl EDA](https://www.kaggle.com/code/kalilurrahman/nfl-data-bowl-2023-offensive-plays-eda/notebook)
